src/runtime/schema.ts

```diff
@@ -160,10 +168,25 @@ export class Schema {
       if (!Schema.typesEqual(fields[name], type)) {
         return false;
       }
+      // this can return null (in the case where we don't know), hence equality with false
```

Comment:
So this suggests an enum would be better..

Reply:
I think this also suggests that the strict null checks option would be valuable (as the type `boolean `and `boolean|null` are then considered different).

I'll try to find time to find out how much effort this would cost.